modules/remotecontrol/Controller.js

```diff
@@ -76,6 +77,7 @@ export default class Controller extends RemoteControlParticipant {
         if(!this.enabled) {
             return Promise.reject(new Error("Remote control is disabled!"));
         }
+        logger.debug("Requsting remote control permissions from: " + userId);
```

Comment:
Why this is logged on debug level ? Will this message appear much more often that the other added in this commit ? For me it would make more sense to have it logged on the "log" level as well.